### PR TITLE
fix: tasklist stringfy request

### PR DESCRIPTION
**Pull request by nathansandi**
## Description

Add stringfy on tasklist variables serch

## When should this change go live?

Fix - Ready to go live

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@nathansandi Is there an epic or task attached for this? I can give you a 👍 review to merge, but I have no idea if this is correct 😅 

Alternatively, someone from the Tasklist team could review it.

**Comment by nathansandi** [comment]
> @nathansandi Is there an epic or task attached for this? I can give you a 👍 review to merge, but I have no idea if this is correct 😅
> 
> Alternatively, someone from the Tasklist team could review it.

Hey @akeller this is only a minor fix on a Task API usage that was making confusion to the customers :) The issue is: https://github.com/orgs/camunda/projects/41/views/2?pane=issue&itemId=46613880

**Review by akeller — APPROVED** [review]
With the linked issue, I can approve this as the change matches 👍 

If you'd prefer to wait for a review from the Tasklist team, you can do that too.

**Pull request merged**
